.github/workflows/release.yml

```diff
@@ -27,6 +27,10 @@ jobs:
         if: steps.branch.outputs.BRANCH_NAME == 'main'
         run: |
           make build-all-small
+      - name: Build Carrier images
+        if: steps.branch.outputs.BRANCH_NAME == 'main'
+        run: |
+          make build-images
```

Comment:
We need to check if we are logged in to Docker at this point

Reply:
Fixed  in https://github.com/SUSE/carrier/pull/260/commits/042e13441ce26224ac00de2f0f083377a76d4ec7